stress/common/src/main/java/alluxio/stress/Parameters.java

```diff
@@ -173,6 +173,15 @@ public String getDescription(List<String> fields) {
     return new Pair<>(new ArrayList<>(commonFields), new ArrayList<>(uniqueFields));
   }
 
+  /**
+   * Notice the function name can't be getOperation since Jackson would transfer this function in to
+   * json value, break Parameter serialization and cause serialization error.
+   * @return the Operation Enum
+   */
+  public Enum<?> operation() {
```

Comment:
The reason I put the operation in GeneralParameters is that not all benchmarks need an operation field. So we may need to check whether this change affect other benchmark

Reply:
There are operation definitions in many parameters, and the way to use them is to directly take the `mOperation` variable (`alluxio.stress.client.ClientIOParameters#mOperation`  etc ). 
In fact, I feel that defining an `operation()` method here can be used in many places. I tested it, if other implementation classes do not implement it, there will be no problem.

> The reason I put the operation in GeneralParameters is that not all benchmarks need an operation field. So we may need to check whether this change affect other benchmark

There are operation definitions in many parameters, and the way to use them is to directly take the `mOperation` variable (`alluxio.stress.client.ClientIOParameters#mOperation`  etc ). 
In fact, I feel that defining an `operation()` method here can be used in many places. I tested it, if other implementation classes do not implement it, there will be no problem.

Reply:
> The reason I put the operation in GeneralParameters is that not all benchmarks need an operation field. So we may need to check whether this change affect other benchmark

There are operation definitions in many parameters, and the way to use them is to directly take the mOperation variable (alluxio.stress.client.ClientIOParameters#mOperation etc ).
In fact, I feel that defining an operation() method here can be used in many places. I tested it, if other implementation classes do not implement it, there will be no problem.